content/posts/scientific-python/cookie/introduction/index.md

```diff
@@ -0,0 +1,204 @@
+at the summit: [scientific-python/repo-review][repo-review] (was
+`scikit-hep/repo-review`) is now a completely general framework for implementing
+checks in Python 3.10+. The checks have been moved to `sp-repo-review`, which is
+now part of scientific-python/cookie. There are too many changes to list here,
+so just the key ones in 0.6, 0.7 & 0.8:
+
+- Extensive, beautiful [documentation](https://repo-review.readthedocs.io) for
+  check authors at (used to help guide the new docs guide page & template
+  update!)
+- Support for four output formats, [rich][] (improved), svg, json (new), html
+  (new).
+- Support for listing all checks.
+- GitHub Actions support with HTML step summary, [pre-commit][] support.
+- Generalized topological sorting to fixtures, dynamic fixtures.
+- Dynamic check selection (via fixtures! Basically everything is powered by
+  fixtures now.)
+- URL support in all output formats (including the terminal and WebApp!)
+- Support for package not at root of repo.
+- Support for running on a remote repo from the command line.
+- Support for select/ignore config in `pyproject.toml` or command line.
+- Pretty printed and controllable sorting for families.
+- Supports running from Python, including inside a readme with something like
+  cog.
+
+The
+[full changelog](https://repo-review.readthedocs.io/en/latest/changelog.html)
+has more - you can even see the 10 beta releases in-between 0.6.x and 0.7.0
+where a lot of this refactoring work was happening. If you have configuration
+you’d like to write check for, feel free to write a plugin!
+
+## sp-repo-review
+
+Finally, sp-repo-review contains the previous repo-review plugins with checks:
+
+- Fully cross-linked with the development guide. Every check has a URL that
+  points to a matching badge inside the development guide where the thing the
+  check is looking for is being discussed!
+- Full list of checks (including URLs), produced by cog, in
+  [readme](https://pypi.org/p/sp-repo-review).
+- Also ships with GitHub Action and [pre-commit][] checks
+- Released (in sync with cookie & guide, as they are in the same repo) as
+  CalVer,
+  [with release notes](https://github.com/scientific-python/cookie/releases).
+- Split CI that selects just want to run based on changed files, with green
+  checkmark that respects skips (based on the excellent contrition to
+  pypa/build).
+
+## Using the guide
+
+If you have a guide, I'd like for you to compare it with the Scientific Python
+Development Guide, and see if we are missing anything - bring it to our
+attention, and maybe we can add it. And then you can link to the centerally
+maintained guide instead of manually maintaining a complete custom guide. See
+[scikit-hep/developer][] for an example; many pages now point at this guide now.
+We can also provide org integrations for the cookiecutter, providing some
+customizations when a user targets your org (targeting `scikit-hep` will add a
```

Comment:
Could we do something similar for `scientific-python` here (i.e., add badges)?

Reply:
Yes, sure. That's the idea, we can offer integration if a specific target org is set. Do you have some in mind?